src/dotty/tools/dotc/sbt/ShowAPI.scala

```diff
@@ -0,0 +1,156 @@
+// This file is copied straight from
```

Comment:
as we always have sbt on classpath, isn't it simpler to simply use the one provided by sbt instead of maintaining a copy of it?